netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
> I am still not sure if showing server version is very helpful and needed. Unlike docker we have different binaries altogether so each binary having their own --version flag shall suffice. But this might be just me.

`netctl version` exercises the REST endpoint to query the version information.